### ISSUE TITLE
FileEventsOutput.py should read a config file

### ISSUE THREAD

**Issue by gaelanadams** [refractionPOINT/limacharlie#57]
Hello, 

FileEventsOutput.py requires a user to modify a code file to change the output_dir or is_flat directive.
This makes it easy for upgrades to go wrong or cause conflicts.

It would be better if LC moved towards a centralized folder for (multiple) configuration files or at least a configuration directory for beach/hcp/. 

This would allow for easier backup of configuration changes as well as reduce the upgrade complexity. 

Thank you,

**Comment by maximelb** [comment]
Agreed the Patrol isn't the right place for that. It used to be placed there because since Beach is distributed on a cluster, having to mirror config files was kind of a pain. I think the right way to deal with those is to port it to use the new(ish) mechanism where we have deployment configs stored in the database and accessible programatically via an RPC to the DeploymentManager actor. It's what we're using for most other recent configs.

If we did that we could make it configurable via the web UI or trivially via a command line call. Adding this to the backlog.
